pkg/controllers/machine_controller.go

```diff
@@ -590,7 +647,7 @@ func (r *MachineReconciler) setDomainMetadata(log logr.Logger, machine *api.Mach
 	return nil
 }
 
-func (r *MachineReconciler) setDomainResources(ctx context.Context, log logr.Logger, machine *api.Machine, domain *libvirtxml.Domain) error {
+func (r *MachineReconciler) setDomainResources(machine *api.Machine, domain *libvirtxml.Domain) error {
```

Comment:
FYI: it isn't very good practice doing cleaning changes with functional changes. I understand it is easier but It little complicates review process (this changes doesn't do anything with Issue) and depends on merge functionality you can have problem with reversing of commit.